### ISSUE TITLE
[Feature request] - Ability to disable presets. 

### ISSUE THREAD

**Issue by tomtom13** [jmcollin78/versatile_thermostat#522]
Request is pretty simple - ability to disable all the preset stuff. Some people have different approach to managing heating and those presets are just confussing non-technical users.

It would be nice to also disable the modes.

All of those in a sense of NOT exposing this to HA so those are not automatically harvested by HASS.

**Comment by tomtom13** [comment]
Yeah, maybe don't close the issue if you've not confirmed with the reporter that this actually work.
You can't set value of 0 if you set minimum allowed temp higher than 0. Sorry, but one setting of temp ranges covering UI and backend configuration is "suboptimal".

**Comment by jmcollin78** [comment]
Sorry to close the issue to rapidly, before 6.3.0 it was possible to specify 0 at temperature which deactivate the preset.

It is now not possible, that is right. So I have no way now to disable a preset.
I will think about this.

**Comment by tomtom13** [comment]
I would suggest a temp setting at the central place - ie "central config" and a secondary min / max setting in thermostat place. That way you can allow global configuration, and then allow people to set per thermostat settings. 
Or simply:
- allow whatever temp user wants for presets
- allow disabling presets via configuration (not through the device or other funky kung-fu)

**Comment by jmcollin78** [comment]
no activity and no real need. I close.

**Comment by tomtom13** [comment]
Yeah, you need to allow minimum temp globally to be 0 - otherwise, you can't set 0 degree for preset to dissable it. I've pointed it out, but was more or less dismissed.

**Comment by jmcollin78** [comment]
> I have been using versatile thermostat for over a year -- thanks for great functionality!
> 
> Now a year later I added another thermostat and tried to set temperature to 0 on two of the presets mode to disable them -- but it looks like they are still present. Is there some other way? For the other thermostats I have two two preset -- comfort and frost.

No it is not possible. A developper starts but stops. This is not so easy with VTherm architecture. Some wants more presets, some wants less presets, ... One user, one need.

The best to do is to make the preset list configurable but this is much complicated for now.
